env-r.yml

```diff
@@ -2,403 +2,432 @@ channels:
   - conda-forge
   - bioconda
 dependencies:
```

Comment:
Can you remake the `env-r.yml` using `--no-builds`? See [here](https://bioconda.github.io/faqs.html#why-shouldn-t-i-include-build-numbers-in-my-environment-yamls) for rationale.

Reply:
Thanks, I noticed no builds in lcdb-wf and was wondering how to get that. Will add